ui/tests/integration/components/replication-dashboard-test.js

```diff
@@ -47,8 +47,22 @@ module('Integration | Enterprise | Component | replication-dashboard', function(
     assert.dom('[data-test-flash-message]').doesNotExist('no flash message is displayed on render');
   });
 
+  test('it updates the dashboard when the replication mode has changed', async function(assert) {
+    await render(hbs`<ReplicationDashboard
+      @replicationDetails={{replicationDetails}}
+      @clusterMode={{clusterMode}}
+      @isSecondary={{isSecondary}}
+    />`);
+
+    assert.dom('[data-test-selectable-card-container="secondary"]').exists();
```

Comment:
Could we also check here that `[data-test-selectable-card-container="primary"]` _doesn't_ exist? And vice versa below